src/AccessibilityInsights.SharedUx/ViewModels/BaseActionViewModel.cs

```diff
@@ -141,13 +141,13 @@ protected virtual void InvokeMethod()
         /// <returns></returns>
         internal static BaseActionViewModel GetActionViewModel(A11yPattern p, MethodInfo m)
         {
-            if (m.ReturnType == typeof(DesktopElement) || m.ReturnType == typeof(List<DesktopElement>))
+            if (m.ReturnType == typeof(DesktopElement) || m.ReturnType == typeof(IList<DesktopElement>))
```

Comment:
Changed for [GetChildren](https://github.com/microsoft/axe-windows/pull/553/files#diff-31258dafbd40185d22311c71844afc772bdfd4f3d7754985aa38687b1fb0ecbfR61) and methods returning a [list of desktop elements](https://github.com/microsoft/axe-windows/pull/553/files#diff-1e18ee3a49cb096ca67f4a4014fd2dd97b6ca36c3021478f1bba0f904b069a07R72)